react-app/src/app/screens/Game/index.js

```diff
@@ -61,12 +47,38 @@ class Game extends Component {
           <Board squares={current.squares} onClick={this.handleClick} />
         </div>
         <div className={style.gameInfo}>
-          <div>{status}</div>
+          <div>{this.props.status}</div>
           <ol>{moves}</ol>
         </div>
       </div>
     );
   }
 }
 
-export default Game;
+Game.propTypes = {
+  history: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.string)),
+  stepNumber: PropTypes.number,
+  xIsNext: PropTypes.bool,
+  status: PropTypes.string,
+  jumpTo: PropTypes.func,
+  toggleMark: PropTypes.func,
+  updateStatus: PropTypes.func
+};
+
+function mapStateToProps(state) {
+  return {
+    history: state.game.history,
+    stepNumber: state.game.stepNumber,
+    xIsNext: state.game.xIsNext,
+    status: state.game.status
+  };
+}
+
+const mapDispatchToProps = {
+  ...gameActions
```

Comment:
You can do `const mapDispatchToProps = gameActions`.
This approach is great, but i cant notice if you understood how mapDispatchToProps works. What if one action needed a different argument? Could you implement mapDispatchToProps as an arrow function?
More info here: https://redux.js.org/basics/usagewithreact#implementing-container-components

Reply:
ok, I did this consciously, but I will implement mapDispatchToProps as an arrow function to show that I understood the concept :wink: